dali/ft/filecopy.cpp

```diff
@@ -2623,6 +2623,49 @@ void FileSprayer::setTarget(IDistributedFile * target)
     }
 }
 
+void FileSprayer::checkTargetPath(RemoteFilename & filename)
+{
+    StringBuffer targetFilePath;
+
+    filename.getRemotePath(targetFilePath);
+    // prune the leading "//" from remote path
+    targetFilePath.remove(0, 2);
+
+    StringBuffer netaddress;
+    filename.queryIP().getIpText(netaddress);
+    const char * pnetaddress = netaddress.str();
+
+    if (streq("127.0.0.1", pnetaddress))
+        throwError1(DFTERR_LocalhostAddressUsed, targetFilePath.str());
+
+    //remove ip address from the beginning of the path
+    targetFilePath.remove(0, netaddress.length());
```

Comment:
Instead of getting the remote path and stripping it back.
It would be cleaner to use getLocalPath() - I believe that will return the local portion of the full filename, but haven't got the source to double check.